src/utils/validation.ts

```diff
@@ -167,6 +167,6 @@ export const isValidSSN = <T>(
   } else if (!/^\d{9}$/.test(ssnString)) {
     return 'Please enter a valid 9 digit Social Security number (dashes allowed)';
   } else {
-    return '';
+    return;
```

Comment:
Can we put this back to `return ''` to keep it consistent with returning a string like the other validation functions?